pkgs/data/misc/cacert/setup-hook.sh

```diff
@@ -1,7 +1,7 @@
-export NIX_SSL_CERT_FILE=@out@/etc/ssl/certs/ca-bundle.crt
+export NIX_SSL_CERT_FILE="${NIX_SSL_CERT_FILE:-@out@/etc/ssl/certs/ca-bundle.crt}"
```

Comment:
Would it make sense to check that the path in the env var is readable first before prioritizing its value over the built one?
I believe this change requires it to now be included in `extra-sandbox-paths`, which is unlikely to be the case in existing deployments using `NIX_SSL_CERT_FILE` prior to this change - and thus will suddenly point to something that doesn't exist as far as the builder is concerned.

Reply:
> Would it make sense to check that the path in the env var is readable first before prioritizing its value over the built one?

Maybe? It wouldn't hurt, but I'm also surprised that you'd need to.

> I believe this change requires it to now be included in extra-sandbox-paths, which is unlikely to be the case in existing deployments using NIX_SSL_CERT_FILE prior to this change

For build-daemon use cases, I've needed to work hard to set NIX_SSL_CERT_FILE. Either add it to the builder's env intentionally, or use the [`inpure-env` config](https://nix.dev/manual/nix/2.22/command-ref/conf-file#conf-impure-env). So I assumed it wouldn't be accidentally set.

Maybe this is more common in a nixos deployment? It might be good to understand your use case and what happens after these changes, to confirm whether the right fix is "ignore NIX_SSL_CERT_FILE when it's not readable", or "don't do that thing you're doing". Maybe raise an issue and link it here for better visibility?